pyvista/core/filters/data_set.py

```diff
@@ -2445,12 +2537,13 @@ def streamlines(dataset, vectors=None, source_center=None,
             source.SetPoint1(pointa)
             source.SetPoint2(pointb)
             source.SetResolution(n_points)
+            _update_alg(source, progress_bar, 'Making Line')
         else:
             source = _vtk.vtkPointSource()
             source.SetCenter(source_center)
             source.SetRadius(source_radius)
             source.SetNumberOfPoints(n_points)
-        source.Update()
+            _update_alg(source, progress_bar, 'Making Point')
         input_source = pyvista.wrap(source.GetOutput())
         output = dataset.streamlines_from_source(input_source, vectors, **kwargs)
```

Comment:
Pass the progress bar arg here as well.

Reply:
Fix in https://github.com/pyvista/pyvista/pull/1519/commits/22305b87045b5da21cf7fc19af405154fc2bd75d